### PR TITLE
fix: function parameter correction in readme

### PR DESCRIPTION
**Pull request by unique-fabian**

### CODE REVIEW

File: unique_sdk/README.md [hunk 1]
```diff
@@ -690,8 +690,8 @@ Allows for uploading files that then get ingested in a scope or a chat.
 
 ```python
 createdContent = upload_file(
-    userId,
-    companyId,
+    companyId=companyId,
```
Comment (sheiksadique) [inline unique_sdk/README.md:693]: Is the order of parameters here critical? If yes, is this order correct now. Looks like it was flipped.

Comment (unique-fabian) [reply unique_sdk/README.md:693]: Now the order is correct.

Comment (sheiksadique) [reply unique_sdk/README.md:693]: Yes order is crucial.

**Review by sheiksadique — APPROVED** [review]
lgtm

**Pull request merged**